_includes/app/txn-sample.py

```diff
@@ -1,20 +1,20 @@
 # Import the driver.
 import psycopg2
+import psycopg2.errorcodes
 
 # Connect to the cluster.
 conn = psycopg2.connect(database='bank', user='maxroach', host='localhost', port=26257)
 
-# Ensure we are running with transactions.
-conn.set_session(autocommit=False)
 
 def onestmt(conn, sql):
     with conn.cursor() as cur:
         cur.execute(sql)
 
+
 # Wrapper for a transaction.
 # This automatically re-calls "op" with the open transaction as an argument
 # as long as the database server asks for the transaction to be retried.
-def txnWrapper(conn, op):
+def run_transaction(conn, op):
```

Comment:
LGTM. 

I'd also like to start thinking about adding more tutorials, one of which could be sqlalchemy-specific, for example. That would allows us to leave these interface-specific and the easiest/quickest way to get a client talking to cockroachdb.